src/storeController/createImportPackage.ts

```diff
@@ -83,8 +84,11 @@ async function reflinkPkg (
   const pkgJsonPath = path.join(to, 'package.json')
 
   if (!opts.filesResponse.fromStore || opts.force || !await exists(pkgJsonPath)) {
-    await mkdirp(to)
-    await execFilePromise('cp', ['-r', '--reflink', from + '/.', to])
+    const staging = `${to}+stage`
+    await mkdirp(staging)
+    await rimraf(to)
+    await execFilePromise('cp', ['-r', '--reflink', from + '/.', staging])
+    await fs.rename(staging, to)
```

Comment:
just use [rename-overwrite](https://github.com/zkochan/rename-overwrite) and you won't need rimraf

Reply:
Rename-overwrite won't work with reflink

Reply:
Or well, I am not sure it would work

Reply:
EDIT: sorry, I'll use it